Tools/autotest/quadplane.py

```diff
@@ -1858,6 +1867,47 @@ def RTL_AUTOLAND_1_FROM_GUIDED(self):
 
         self.fly_home_land_and_disarm()
 
+    def RealFlightTitanHover(self):
+        '''Test the RealFlight Titan Cobra'''
+        frame = "flightaxis"
+        if self.realflight_address is not None:
+            frame += f":{self.realflight_address}"
+        self.customise_SITL_commandline(
+            [
+                "--home", "40.0594626,-88.5513292,206.0,0",
+                "--wipe",
```

Comment:
wipe must be done as a keyword argument  to `customise_SITL_commandline` or bad things happen